Makefile.in

```diff
@@ -37,6 +37,7 @@ build_mode=@build_mode@
 
 INSTALL_PROGRAM=install -m 755
 INSTALL_DATA=install -m 644
+PWD?=$(shell pwd)
```

Comment:
The environment variable `PWD` seams to be missing while executing the main `Makefile` on AppVeyor with MSys2/MinGW64 environment/tool chain.

So if `PWD` is missing, use the shell command `pwd` and assign it.